install/linux/docker-ee/oracle.md

```diff
@@ -6,90 +6,90 @@ redirect_from:
+- Remove old Docker repos from `/etc/yum.repos.d/`.
+- Disable SELinux if installing or upgrading Docker EE 17.06.1.
 
-{% include ee-linux-install-reuse.md section="ee-url-intro" %}
+### Architectures and storage drivers
 
-### OS requirements
+Docker EE supports {{ linux-dist-long }} 64-bit 7.3 or higher, running the Red Hat Compatible kernel (RHCK) 3.10.0-514 or higher. Older versions of {{ linux-dist-long }} are not supported.
 
-To install Docker EE, you need the 64-bit version of {{ linux-dist-long }} 7.3
-or higher, running the Red Hat Compatible kernel (RHCK) 3.10.0-514 or higher.
-Older versions of {{ linux-dist-long }} are not supported.
+On {{ linux-dist-long }}, Docker EE only supports the `devicemapper` storage driver. In production, you must use it in `direct-lvm` mode, which requires one or more dedicated block devices. Fast storage such as solid-state media (SSD) is recommended. Do not start Docker until properly configured per the [storage guide](/storage/storagedriver/device-mapper-driver/){: target="_blank" class="_" }.
 
-In addition, you must use the `devicemapper` storage driver if you use
-Docker EE. On production systems, you must use `direct-lvm` mode, which
-requires one or more dedicated block devices. Fast storage such as solid-state
-media (SSD) is recommended.
+### Find your Docker EE repo URL
 
-> **Docker EE cannot install on {{ linux-dist }} with `selinux` enabled!**
->
-> If you have `selinux` enabled and you attempt to install Docker EE 17.06.1,
-> you get an error that the `container-selinux` package cannot be found.
-{:.warning }
+{% include ee-linux-install-reuse.md section="find-ee-repo-url" %}
 
-### Uninstall old versions
+### Uninstall old Docker versions
 
-Older versions of Docker were called `docker` or `docker-engine`. If these are
-installed, uninstall them, along with associated dependencies.
+The Docker EE package is called `docker-ee`. Older versions were called `docker` or `docker-engine`. Uninstall all older versions and associated dependencies. The contents of `/var/lib/docker/` are preserved, including images, containers, volumes, and networks.
 
 ```bash
 $ sudo yum remove docker \
                   docker-engine \
                   docker-engine-selinux
 ```
 
-It's OK if `yum` reports that none of these packages are installed.
-
-The contents of `/var/lib/docker/`, including images, containers, volumes, and
-networks, are preserved. The Docker EE package is now called `docker-ee`.
+## Repo install and upgrade
 
-## Install Docker EE
+{% include ee-linux-install-reuse.md section="using-yum-repo" %}
 
-{% include ee-linux-install-reuse.md section="ways-to-install" %}
-
-### Install using the repository
-
-Before you install Docker EE for the first time on a new host machine, you need
-to set up the Docker repository. Afterward, you can install and update Docker EE
-from the repository.
+{% capture selinux-warning %}
+> Docker EE cannot install on {{ linux-dist-long }} with SELinux enabled!
```

Comment:
I think we should use the same convention for note titles as we use for regular headings: no punctuation at the end.